e2e/portal/tests/ViewPayments/ExportPaymentsWithDateRange.spec.ts

```diff
@@ -14,15 +14,10 @@ import PaymentsPage from '@121-e2e/portal/pages/PaymentsPage';
 
 // Get current date information
 const currentDate = new Date();
-const day = currentDate.getDate();
-const month = currentDate.getMonth();
-const year = currentDate.getFullYear();
-const startDate = `${year}-${month}-${day}`;
-const endDate = `${year}-${month}-${day}`;
-const startDateFuture = `${year}-${month}-${day + 1}`;
-const endDateFuture = `${year}-${month}-${day + 1}`;
-const startDatePast = `${year}-${month}-${day - 1}`;
-const endDatePast = `${year}-${month}-${day - 1}`;
+const futureDate = new Date(currentDate);
+futureDate.setDate(currentDate.getDate() + 1);
```

Comment:
```suggestion
const futureDate = addDays(currentDate, 1);
```

We could/should also use ~~some library~~[`date-fns`](https://date-fns.org/) for these kind of things..

Reply:
Sure. I will make a comment in the code.